TakingITGlobal/commit2act
# [TIG-500] Page level tests

## Ticket

* [TIG-500](https://app.clickup.com/t/9009201449/TIG-500)

## Description

This PR adds basic tests for some high-level page components, including:

* `AccountSettings`
* `Actions`
* `AdminDashboard`
* `ValidateActions`

It also expands tests for `AppNav`, sets up reusable test utilities and mocking in `utils/jest-mock-utils.js`, and makes small a11y fixes throughout pages and their sub-components as caught by the `jest-axe` tool. 

This PR does not include tests for all pages, as the impact across multiple files and sub-files was growing more than seemed responsible. It does, however, setup patterns and practices which can be used by any other devs to help expand the coverage!

## Testing

* Pull down and install the branch
* From the root, run `npm run test`
* Confirm all tests pass
* Build the app locally using `npm start`
* Peruse the impacted pages in-browser to confirm no functionality has changed

## Screenshots

**Passing tests output in Terminal:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/20a6e6a2-e4c9-41c6-b7f2-5700b11c2aca)
